### ISSUE TITLE
questions about config file

### ISSUE THREAD

**Issue by soobin419** [ewrfcas/MAE-FAR#13]
Hello.
Thanks for your great work! I am a beginner in this field.

I have some questions as below.
1. Is it correct to sample 36,500 (same as the number of verification data) of Coco Mask and Irregular Mask and use them as test_mask_flist ?
![image](https://user-images.githubusercontent.com/62980764/231683289-37c01617-f0db-4bb5-b6df-6a5b7ce8ceaa.png)

2. Is it correct to modify the val_flist, test_path, and test_mask_flist parts of the configuration file to match the test data provided on the Palces2 official site when testing after training is completed? There are 36,500 images in validation data and 328,500 images in test data. So I understood that there should be 36,5000 images and masks in val_flist and test_mask_flist during training, and 328,500 images and masks in val_flist and test_mask_flist during testing(Assuming that all data is used). Is my understanding correct? Or did you use the same 36,500 data for validation and test?
![image](https://user-images.githubusercontent.com/62980764/231684845-55ea27f2-efc3-4489-b795-ef39d0f43408.png)

3. I set the sample size of the config file to 12. But there are only 8 images in the image saved in the sample folder as shown below. What does sample size mean?
![image](https://user-images.githubusercontent.com/62980764/231768888-50bd8dd9-ffe7-4235-99e5-1e43c135ae44.png)

4. You mention that 'every 200k steps in Places2 halves the learning rate every 100k steps in FFHQ' in the paper, but Decay_steps in the config file is set to [600000, 700000, 800000]. Should I change it to 200k step interval?
![image](https://user-images.githubusercontent.com/62980764/231816908-6749f28a-891f-49ef-bdc1-0b8ad43438af.png)

**Comment by ewrfcas** [comment]
Hi,

1. Yes. 
2. We only used the 36500 validation.
3. Sorry that the sample_size is not working now. The real sample size is simply according to your training batch for a single GPU.
4. The re-implementation is slightly different from the one in the paper. We found that this decay worked better.

**Comment by soobin419** [comment]
OK, Thank you !!
